Description: Fix permission on metronome.version
Author: Julien Malik <julien.malik@paraiso.me>

--- metronome.orig/Makefile
+++ metronome/Makefile
@@ -46,7 +46,7 @@
	install -m755 ./generate_log.install $(MODULES)/muc_log_http/generate_log
	install -m644 certs/* $(CONFIG)/certs
	test -e $(CONFIG)/metronome.cfg.lua || install -m644 metronome.cfg.lua.install $(CONFIG)/metronome.cfg.lua
-	test -e metronome.version && install metronome.version $(SOURCE)/metronome.version || true
+	test -e metronome.version && install -m644 metronome.version $(SOURCE)/metronome.version || true
 	$(MAKE) install -C util-src
 
 git-upgrade:
